code/lib/builder-manager/package.json

```diff
@@ -54,6 +54,7 @@
     "ejs": "^3.1.8",
     "esbuild": "^0.16.4",
     "esbuild-plugin-alias": "^0.2.1",
+    "esbuild-plugin-environment": "^0.2.2",
```

Comment:
I published version `0.2.3` of this plugin.
If you update to that version, you will get rid of some transitive dependencies.